### ISSUE TITLE
Design Pattern - max 10 points

### ISSUE THREAD

**Issue by Zighome24** [Zighome24/CS2340_GoGreek#14]
From TA: 
Design Patterns (1 - 10 points). For each different design pattern that students used they can get points. Points vary based on difficulty:

1 point Singleton, 5 points Command Pattern, 3 points Factory, 10 points is reserved for teams that integrated their design patterns really well into a superior design. I saw a couple of teams that did full MVP pattern with interfaces 7 - 10 points (not for teams that just did a little MVC), Facade 2 points.

Basically, if we use a proper set up for the ways models and controllers are used then we should be gucci mane.
